WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureOrchestrator.swift

```diff
@@ -202,20 +186,20 @@ private extension PaymentCaptureOrchestrator {
             }
 
             switch result {
-            case .success:
-                self.celebrate() // plays a sound, haptic
-                self.saveReceipt(for: order, params: receiptParameters)
-                onCompletion(.success(receiptParameters))
-            case .failure(let error):
-                onCompletion(.failure(error))
-                return
+                case .success:
+                    self.celebrate() // plays a sound, haptic
+                    self.saveReceipt(for: order, params: receiptParameters)
+                    onCompletion(.success(receiptParameters))
+                case .failure(let error):
+                    onCompletion(.failure(error))
+                    return
```

Comment:
super nit: I felt like the changes between L122 and L195 can be reverted since these are using different styles than in the codebase 🤔

Reply:
Reverted - https://github.com/woocommerce/woocommerce-ios/pull/6519/commits/7a94206c64ed11afad89a1c5a0e13e1222df068d